newapi/ooniapi/probe_services.py

```diff
@@ -183,12 +186,14 @@ def check_in():
         log.error(str(e), exc_info=1)
         conf = {}
 
+    utc_time = datetime.utcnow().strftime('%Y%m%dT%H%M%S')
```

Comment:
The format of this timestamp does not match the docstring. I would suggest, for consistency with other places where we have timestamp, to use the format: `YYYY-mm-ddTHH:MM:SSZ`

Reply:
Fixed